concepts/symbols/about.md

```diff
@@ -0,0 +1,80 @@
+# About
+
+[Symbols][symbols] are named identifiers that can be used to refer to a value.
+Symbols are created through a symbol literal, which is by prefixing a name with a `:` character, e.g. `:foo`.
+They also allow for being written as strings, e.g. `:"foo"`, which allows for example spaces in the name.
+
+```ruby
+:foo # => :foo
+:"foo boo" # => :"foo boo"
+```
+
+Symbols are used in many places in the language, including as keys in hashes, to represent method names and variable names.
+
+## Identifier
+
+What makes symbols different from strings is that they are identifiers, and not data or text.
+This means that two symbols with the same name are always the same object.
+
+```ruby
+"foo".object_id # => 60
+"foo".object_id # => 80
+:foo.object_id # => 1086748
+:foo.object_id # => 1086748
+```
+
+## Modifying Symbols
+
+Symbols are immutable, which means that they cannot be modified.
+This means that when you "modify" a symbol, you are actually creating a new symbol.
+There are a few methods that can be used to modify symbols, but they all return new symbols.
+All methods can be found in the [Symbol API][symbols-api].
+
+```ruby
+:foo.upcase # => :FOO
+
+:foo.object_id # => 1086748
+:foo.upcase.object_id # => 60
+```
+
+The benefit of symbols being immutable is that they are more memory efficient than strings, but also safer to use as identifiers.
+
+## Conversion
+
+Symbols can be converted to strings and vice versa.
```

Comment:
How do we state this without suggesting that they are really converted, or modified, to be a string?

Reply:
I think conversion is the correct wording here since I think it is what a student links with getting another type.
Since every "conversion" when you go from one type to another, isn't actually a "conversion" in the sense that the "converted" value is the same as the original.

```irb
irb(main):001:0> a = 5
=> 5
irb(main):002:0> a.object_id
=> 11
irb(main):003:0> a.to_s.object_id
=> 26870
irb(main):004:0> a.object_id
=> 11
```

And these methods `to_s` and `to_sym` is put under the section of "Methods for Converting", in the official doc.

![image](https://github.com/exercism/ruby/assets/69751659/c97e2bdf-824c-46f8-9454-225bd47b1053)

Reply:
I think "coerce" is the functionality of conversion from a symbol to something else, so yeah, but definitely not modify.  "conversion" is nuanced, since I can convert a hard top vehicle to a soft top vehicle, and it is the same vehicle, where here we would get a brand new object, instead of a true "conversion".

What you said about the conversion not really being a true conversion.  As long as we avoid the idea that modification is going on, I do think that "convert" and "conversion" is OK.  We should accentuate that it is a new object, though.  (Ruby is strongly typed, so having the same object change types would be weird.)